src/naive/datetime/tests.rs

```diff
@@ -1,9 +1,37 @@
 use super::NaiveDateTime;
 use crate::oldtime::Duration;
 use crate::NaiveDate;
-use crate::{Datelike, FixedOffset, Utc};
+use crate::{Datelike, FixedOffset, Timelike, Utc};
 use std::i64;
 
+#[test]
+fn test_datetime_from_timestamp_millis() {
+    let timestamp_millis: i64 = 1662921288000; //Sunday, September 11, 2022 6:34:48 PM
+    let naive_datetime = NaiveDateTime::from_timestamp_millis(timestamp_millis);
+    assert!(naive_datetime.is_some());
+    assert_eq!(timestamp_millis, naive_datetime.unwrap().timestamp_millis());
+    assert_eq!(naive_datetime.unwrap().year(), 2022);
+    assert_eq!(naive_datetime.unwrap().month(), 9);
+    assert_eq!(naive_datetime.unwrap().day(), 11);
+    assert_eq!(naive_datetime.unwrap().hour(), 18);
+    assert_eq!(naive_datetime.unwrap().minute(), 34);
+    assert_eq!(naive_datetime.unwrap().second(), 48);
+    assert_eq!(naive_datetime.unwrap().timestamp_subsec_nanos(), 0);
```

Comment:
Maybe cover most of the fields with a single `assert_eq!()` against a string-encoded timestamp? Would be a little more compact and readable.

Reply:
good call - I've switched to comparing against a formatted string